src/thor/worker.cc

```diff
@@ -96,6 +96,16 @@ thor_worker_t::thor_worker_t(const boost::property_tree::ptree& config,
 thor_worker_t::~thor_worker_t() {
 }
 
+std::string serialize_to_pbf(Api& request) {
+  std::string buf;
+  if (!request.SerializeToString(&buf)) {
+    LOG_ERROR("Failed serializing to pbf in Thor::Worker - trace_route");
```

Comment:
We're calling this helper from `optimized_route`, `route` & `trace_route`. So maybe we can be remove the `trace_route` part of the error message since it could be generated for other options.